### PR TITLE
added HCW analyses to temporal trends

### PR DESCRIPTION
**Pull request by kya2**
Added HCW analyses. Due to small n, I included only analyses of totals by month (not week) and by sous-coordination (not down to zones).

I need the code for e-accent-aigu please.

### CODE REVIEW

**Comment by thibautjombart** [comment]
@kya2 useful tip: you can link an issue by copy-pasting its url, so you can say something like:

addresses the HCW addition requestsed in https://github.com/reconhub/report_factories_templates/issues/4

File: linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd [hunk 1]
```diff
@@ -226,7 +227,7 @@ For each proportion, we derive the exact binomial confidence interval.
 
 We will rely on `adorn_totals` for computing totals. This approach needs to take into account an
 [issue](https://github.com/sfirke/janitor/issues/285) with `adorn_totals`, which
-ignores the first column of data assuming it is a grouping factor. This will no
+ignores the first column of data assuming it is a grouping factor. This will not
```
Comment (thibautjombart) [inline linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd]: thx for fixing the typos! :)

File: linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd [hunk 2]
```diff
@@ -1269,6 +1269,246 @@ table_community_deaths_top_zones_time %>%
...
+      
+## step 1
+hcw_overall <- x %>%
+  count(hcw) %>%
+  spread(hcw, n, fill = 0, drop = FALSE) %>%
+  mutate(total = nrow(x), # avoid adorn_totals
+        denom = total - unknown,
+        p = prop_to_perc(hcw / denom),
+        p_lower = prop_ci(hcw, denom, "lower", perc = TRUE),
+        p_upper = prop_ci(hcw, denom, "upper", perc = TRUE))
```
Comment (thibautjombart) [inline linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd]: cosmetic, but indentation is off by one space here

File: linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd [hunk 3]
```diff
@@ -1269,6 +1269,246 @@ table_community_deaths_top_zones_time %>%
...
+      
+## step 2
+hcw_overall_time <- x %>%
+  count(month_report, hcw) %>%
+  spread(hcw, n, fill = 0, drop = FALSE) %>%
+  adorn_totals(c("col"), name = "total") %>% 
+  mutate(denom = total - unknown,
+        p = prop_to_perc(hcw / denom),
+        p_lower = prop_ci(hcw, denom, "lower", perc = TRUE),
+        p_upper = prop_ci(hcw, denom, "upper", perc = TRUE))
```
Comment (thibautjombart) [inline linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd]: indentation off by 1

File: linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd [hunk 4]
```diff
@@ -1269,6 +1269,246 @@ table_community_deaths_top_zones_time %>%
...
+      
+      
+## step 3
+ggplot(hcw_overall_time, aes(x = month_report, y = p)) +
+        theme_bw() +
+        geom_ribbon(aes(ymin = p_lower, ymax = p_upper),
+              alpha = .3, fill = color_hcw) +
+        geom_point(color = color_hcw) +
+        geom_line(color = color_hcw) +
+        scale_y_continuous(limits = c(0, 25)) +
```
Comment (thibautjombart) [inline linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd]: avoid hard-coded limits for axes, as these may not be adapted in future updates (unless we have an objective fixed scale)

File: linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd [hunk 5]
```diff
@@ -1269,6 +1269,246 @@ table_community_deaths_top_zones_time %>%
...
+## step 3
+ggplot(hcw_sub_coordination, aes(x = sous_coordination, y = p)) +
+  theme_bw() +
+  geom_errorbar(aes(ymin = p_lower, ymax = p_upper),
+          color = color_hcw) +
+  geom_point(color = color_hcw, size = 2) +
+  geom_label(aes(y = p_upper + 10, label = paste0("n=", denom))) +
+  large_txt +
+  coord_flip() +
+  scale_y_continuous(breaks = seq(0, 100, by = 20),
```
Comment (thibautjombart) [inline linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd]: Hard-coded y axis range is fine here, as these are always %

File: linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd (hunk 4 above)
Comment (kya2) [reply linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd]: I chose this in order to see the downward trend. HCWs should be a very low percentage.

**Pull request merged**